core/domain/exp_jobs_one_off.py

```diff
@@ -470,3 +473,61 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class ExplorationContentValidationJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the html content of an exploration and validates it.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        err_dict = {}
+
+        with open('./core/domain/htmlValidation.yaml', 'r') as fp:
```

Comment:
For this, it seems like it would be useful and cleaner to use certain functions from `utils.py`, such as `dict_from_yaml` and `get_file_contents`?

Reply:
Also, very minor nit: I suggest storing the filepath in a named constant variable in keeping with the style of other code.

Reply:
Done!